forc/src/utils/defaults.rs

```diff
@@ -49,26 +49,64 @@ path = "tests/harness.rs"
 }
 
 pub(crate) fn default_program() -> String {
-    r#"script;
+    r#"contract ;
 
-fn main() {
+    abi MyContract {
 
-}
-"#
-    .into()
-}
+    }
 
-pub(crate) fn default_test_program() -> String {
-    r#"
+    impl MyContract for Contract {
 
-#[tokio::test]
-async fn harness() {
-    assert_eq!(true, true);
-}
+    }
 "#
     .into()
 }
 
+pub(crate) fn default_test_program(project_name: &str) -> String {
+    format!(
+        "{}{}{}{}{}",
+        r#"
+    use fuel_tx::{ContractId, Salt};
+    use fuels_abigen_macro::abigen;
+    use fuels_contract::{contract::Contract, parameters::TxParameters};
+    use fuels_signers::util::test_helpers;
+    
+    // Load abi from json
+    abigen!(MyContract, "./"#,
+        project_name,
+        r#".json");
+    
+    async fn get_contract_instance() -> (MyContract, ContractId) {
+    
+        // Build the contract
+        let salt = Salt::from([0u8; 32]);
+        let compiled = Contract::load_sway_contract("./out/debug/"#,
+        project_name,
+        r#".bin", salt).unwrap();
+    
+        // Launch a local network and deploy the contract
+        let (provider, wallet) = test_helpers::setup_test_provider_and_wallet().await;
+    
+    
+        let id = Contract::deploy(&compiled, &provider, &wallet, TxParameters::default()).await.unwrap();
+    
+        let instance = MyContract::new(id.to_string(), provider, wallet);
+    
+        (instance, id)
+    
```

Comment:
Should probably run the generated Rust template and the Sway template through a formatter, then use that formatted code as the template. Or just manually format these templates.

Reply:
The template `harness.rs` should be formatted now